applications/nrf_desktop/src/modules/Kconfig.ble

```diff
@@ -123,6 +130,22 @@ config DESKTOP_BLE_SCANNING_ENABLE
 	help
 	  Enable device to scan for peripheral devices.
 
+config DESKTOP_BLE_SCAN_START_TIMEOUT
```

Comment:
What do you think about

DESKTOP_BLE_SCAN_START_TIMEOUT_S

and

DESKTOP_BLE_SCAN_DURATION_S

?

Reply:
Ok, I may change it.